builtin/credential/cert/backend_test.go

```diff
@@ -383,3 +384,122 @@ func testConnState(t *testing.T, certPath, keyPath, rootCertPath string) tls.Con
 	connState := serverConn.(*tls.Conn).ConnectionState()
 	return connState
 }
+
+func Test_Renew(t *testing.T) {
+	storage := &logical.InmemStorage{}
+
+	lb, err := Factory(&logical.BackendConfig{
+		System: &logical.StaticSystemView{
+			DefaultLeaseTTLVal: 300 * time.Second,
+			MaxLeaseTTLVal:     1800 * time.Second,
+		},
+		StorageView: storage,
+	})
+	if err != nil {
+		t.Fatal("error: %s", err)
```

Comment:
Is there a reason why other errors are not like this?
They are just t.Fatal(err).